### PR TITLE
Section abou the RoBERTa Base OpenAI detector

### PR DESCRIPTION
**Pull request by 1Maxnet1**

### CODE REVIEW

**Review by LeviUniAcc — APPROVED** [review]
good so far, do you want to write a bit about how it works or is this even public?, if not maybe we write that?

File: paper.tex [hunk 1]
```diff
@@ -54,6 +54,9 @@ \subsubsection{Image detector models}
 
 \subsubsection{Text detector models}
 
+The \textbf{RoBERTa Base OpenAI Detector} is the output detector model of GPT-2. It was obtained by fine-tune a RoBERTa base model with the ouputs of the GPT-2 model containing 1,5 Billion parameters. RoBERTa Base OpenAI Detector can be used to classify text, on whether it is likely to be created by the GPT-2 model. It was published by OpenAI, when the weights of the GPT-2 model 1,5 Billion parameter version was released.
```
Comment (RaphaW) [inline paper.tex]:
```suggestion
The \textbf{RoBERTa Base OpenAI Detector} is the output detector model of GPT-2. It was obtained by fine-tuning a RoBERTa base model with the ouputs of the GPT-2 model containing 1,5 Billion parameters. A RoBERTa Base OpenAI Detector can be used to classify texts, on whether or not it is likely to be created by the GPT-2 model. It was published by OpenAI, when the weights of the GPT-2 model 1,5 Billion parameter version was released.
```

Comment (EhrenbruderRoschtbraten) [reply paper.tex]:
```suggestion
The \textbf{RoBERTa Base OpenAI Detector} is the output detector model of GPT-2. It was obtained by fine-tuning a RoBERTa base model with the outputs of the GPT-2 model containing 1,5 Billion parameters. The RoBERTa Base OpenAI Detector can be used to classify texts, on whether they are likely to be created by the GPT-2 model. It was published by OpenAI, when the weights of the GPT-2 model 1,5 Billion parameter version was released.
```

**Review by EhrenbruderRoschtbraten — CHANGES_REQUESTED** [review]
Looks good content-wise, only a small comment

**Review by EhrenbruderRoschtbraten — APPROVED** [review]

**Review by RaphaW — APPROVED** [review]
LGTM

**Pull request merged**